data/org.freedesktop.UDisks2.xml

```diff
@@ -1344,8 +1344,8 @@
         Gets a file descriptor for the device that is suitable to be
         used for benchmarking the device (transfer rate, access time
         etc.). Note that the file descriptor may be opened with the
-        <literal>O_DIRECT</literal> and <literal>O_SYNC</literal>
-        flags so care must be taken to only perform page-aligned I/O.
+        <literal>O_DIRECT</literal> flag so care must be taken to only perform
+        block-aligned (512 byte aligned) I/O.
```

Comment:
As far as I understand the block size the device specifies does not matter since we talk to the device directly. So in this case the linux block size matters which happens to be 512 byte. If there's no specific number how is a user supposed to implement anything correctly if he doesn't know how to determine the block size / where it comes from?

Why is it worth to mention the `O_SYNC` flag? Is there any extra care that has to be taken because of that flag? Anyone who's interested in the exact flags can always look at the source code.